metal/v1/api_devices.go

```diff
@@ -359,19 +361,19 @@ func (a *DevicesApiService) CreateDeviceExecute(r ApiCreateDeviceRequest) (*Find
 }
 
 type ApiCreateIPAssignmentRequest struct {
-	ctx                       context.Context
-	ApiService                *DevicesApiService
-	id                        string
-	createIPAssignmentRequest *CreateIPAssignmentRequest
+	ctx               context.Context
+	ApiService        *DevicesApiService
+	id                string
+	iPAssignmentInput *IPAssignmentInput
```

Comment:
I'm curious about the objects that were renamed from Create*Request to *Input.
I assume this was an upstream change? If the name happened to be *CreateRequest the objects might match packngo. (just an observation, not necessarily a goal)

Reply:
That's correct, this name change happened because the new spec merge process maintains existing `$ref`s instead of replacing each `$ref` with a more-specifically-named replica.  The old merge process would replace `IPAssignmentInput` with an identical `CreateIPAssignmentRequest` because it is used as the request body of the `createIPAssignment` operation (if there were an `updateIPAssignment` operation that used the same input, the old process would have created identical `CreateIPAssignmentRequest` and `UpdateIPAssignmentRequest` types).  Renaming `IPAssignmentInput.yaml` to `CreateIPAssignmentRequest.yaml` would change the name, although `packngo` doesn't necessarily have an analogue for all of the `*Request` types that are deleted in this PR.